src/PensionsRegulatorApi/Data/SqlOrganisationRepository.cs

```diff
@@ -60,5 +60,62 @@ public IEnumerable<Organisation> GetOrganisationsForPAYEReference(string payeRef
 
             return retrievedOrganisations.AsReadOnly();
         }
+
+        public IEnumerable<Organisation> GetOrganisationsForPAYEReferenceAndAORN(string payeReference, string aorn)
```

Comment:
Can you have a look at refactoring the method?  It feels too long.

Reply:
Done some work on this and reduced a fair amount of repetition.